cdap-data-fabric/src/main/java/io/cdap/cdap/data2/metadata/lineage/field/FieldLineageInfo.java

```diff
@@ -172,6 +176,10 @@ private void computeAndValidateFieldLineageInfo(Collection<? extends Operation>
             connections.add(transform);
           }
           allOrigins.addAll(origins);
+          if (transform.getInputs().size() > transform.getOutputs().size()) {
```

Comment:
Is this correct? Fields can still be dropped even the output is larger than input if there is new fields being generated with old fields dropped.